packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -45,6 +45,7 @@ function createRootStyles(themeVariables?: ThemeVariables) {
     core: css`
       @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
       :root {
+        --w3m-modal-width: 360px;
```

Comment:
did we check with design that this is ok? just wondering if designs expect a fixed width

Reply:
@tomiir Yeah, we use 360px as modal width. The thing I did is moving it to variable instead using in hard code value in CSs. 

Using variable is good when you have to use modal width in different places. I.e: I'm calculating the `max-width` of tooltip depending on the modal width:

```css
    max-width: calc(var(--w3m-modal-width) - var(--wui-spacing-xl));
```